src/pages/docs/general/concepts/cli-tips.mdx

```diff
@@ -0,0 +1,265 @@
+
+```yaml
+scripts:
+ server: npm start
+```
+
+Run `rivet run server` to start the server.
+
+**Options**
+
+By default, it will run the command with a [development token](/docs/general/conceps/dev-tokens) in the staging namespace.
+
+You can change the namespace with `rivet run server --namespace prod`.
+
+You can issue a [public token](/docs/general/concepts/handling-game-tokens) to test against servers running on Rivet with `rivet run server --rivet-servers`.
+
+### `rivet exec`
+
+The `rivet exec` command alows you to run a command with the `RIVET_TOKEN` and `RIVET_API_ENDPOINT` environment variables. For example:
+
+```bash
+rivet exec "npm start"
+```
+
+This also supports the same options as `rivet run` (see above).
+
+## Configuration
+
+### Namespace overrides with `rivet.{ns}.yaml`
+
+To override config properties, you can create YAML configs specific to a namespace.
+
+For example, if you have a `rivet.yaml` file like:
+
+```yaml
+matchmaker:
+ max_players: 32
+ tier: basic-1d2
+ # ..etc..
+```
+
+You can create a file called `rivet.prod.yaml` to override properties for the `prod` namespace. For example, to override the `tier` property:
+
+```yaml
+matchmaker:
+  tier: basic-1d1
+```
+
+### Validate a config
+
+Use the `rivet config validate` command to validate a version config before deploying it. This is useful for catching errors before deploying.
+
+
+### YAML, JSON, and TOML support
+
+Rivet supports multiple config file formats. You can use `rivet.yaml`, `rivet.json`, and `rivet.toml` for your config files.
+
+### Unlink game
+
+To unlink a game, use the `rivet unlink` command. This allows you to run `rivet init` to link a different game.
+
+## Continuous Integration
+
+### Generating GitHub Actions deploy workflow
+
+To generate a GitHub Actions deploy workflow that parallelizes building Docker images & CDN sites, use the `rivet ci generate github` command.
+
+## Deploying
+
+
+### Deploy with config overrides
+
+To override properties in a version you'll deploy, you can use the `--override` (or `-o` for short) flag with the JSON value. This flag can be used multiple times to override multiple properties.
+
+This example only works under POSIX-compliant shells (e.g. bash, zsh, etc., not PowerShell):
+
+```bash
+rivet deploy prod -o 'matchmaker.docker.env.MY_ENV_VAR="foo"' -o 'cdn.build_env.MY_BUILD_ARG="bar"'
+```
+
+The `--override` flag also works with `rivet config validate`.
+
+### Deploying without applying to namespace
+
+To deploy a version without applying it to a namespace, use the `--no-namespace` flag. This is useful for deploying a version to a namespace at a later time.
+
+1. `rivet deploy --no-namespace`
+2. `rivet namespace set-version <namespace> <version>` (this can also be done via Rivet Hub under _Manage Versions_)
+
+## Matchmaker & Docker
+
+### Reuse a Docker image
+
+You can re-use a deployed CDN site by overriding the `docker.image_id` property to prevent building it again.
```

Comment:
@NathanFlurry lol i overlooked my own suggestion, accidentially left the CDN site bit


```suggestion
You can re-use a deployed Docker image by overriding the `docker.image_id` property to prevent building it again.
```